### ISSUE TITLE
wwwapps.ups.com is blocked by uBlock Origin Lite

### ISSUE THREAD

**Issue by eladts** [uBlockOrigin/uAssets#15069]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`wwwapps.ups.com`

### Describe the issue

uBlock Origin **Lite** completely blocks access to the UPS package tracking site.

### Screenshot(s)

![image](https://user-images.githubusercontent.com/6475871/192661143-33afd24a-1fb7-44ba-9a06-0d5736cc9a1a.png)

### uBlock Origin version

0.1.22.9205

### Browser name and version

Chrome 105.0.5195.112

### Settings

none

**Comment by gwarser** [comment]
Select a region -> Select "Tracking" on top -> select "Track package".

**Comment by okiehsch** [comment]
Yes, I can reproduce using uBO-lite, I have no way of fixing it with a filter, so I will transfer it to [uBlock-issues](https://github.com/uBlockOrigin/uBlock-issues)
